codecov.yml

```diff
@@ -0,0 +1,17 @@
+codecov:
```

Comment:
Changed in Defaults but not releases. This will limit the number of checks on PRs to just `changes`

```
codecov:
  branch: master

coverage:
  precision: 2
  round: down
  range: 70...100

  status:
    project: no
    patch: yes

comment: 'off'

```

Reply:
The rest of what's in that codecov config has evidently been deprecated / changed since last we looked at it.

Reply:
Done